src/services/swap_service.ts

```diff
@@ -505,7 +507,8 @@ export class SwapService {
             includePriceComparisons,
         } = params;
         let _rfqt: RfqtRequestOpts | undefined;
-        const isAllExcluded = Object.values(ERC20BridgeSource).every(s => excludedSources.includes(s));
+        const isAllExcluded =
+            excludedSources && Object.values(ERC20BridgeSource).every(s => excludedSources.includes(s));
```

Comment:
excludedSources can be undefined? I thought we set the defaults and it is defaulted to `[]`

Reply:
We had some intermediate parameter type definitions where it was an optional parameter (even though it was always passed). I have now updated it to be non-nullable.